src/CommonAssemblyInfo.cs

```diff
@@ -3,9 +3,9 @@
 using System.Resources;
 using System.Runtime.InteropServices;
 
-[assembly: AssemblyVersion("2.1.0")]
-[assembly: AssemblyFileVersion("2.1.0")]
-[assembly: AssemblyInformationalVersion("2.1.0")]
+[assembly: AssemblyVersion("2.2.0")]
+[assembly: AssemblyFileVersion("2.2.0")]
+[assembly: AssemblyInformationalVersion("2.2.0")]

```

Comment:
Mostly to make it less error-prone; if the version is only in one place, you can't change it somewhere and forget to change it somewhere else...